HHS/Head-Start-TTADP
# [TTAHUB-359] add filters to regional dashboard

## Description of change
Add filters to the regional dashboard page
- Add new filter options 
- Condense configuration for Filter Menu/Filter pills
- Make filters in filter menu configurable
- Filters are reflected in the URLs
- Mark up improvements

I am creating the state filter, originally part of this Jira, separately as it requires its own logic.

## How to test
Add filters to the dashboard. Check that all the behaviors are as expected. Check that the data is updated properly.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-359


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
